addons/mass_mailing_sms/report/mailing_trace_report_views.xml

```diff
@@ -1,5 +1,20 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
+    <record id="mailing_trace_report_sms_view_pivot" model="ir.ui.view">
```

Comment:
You could do a view that inherit from the base one with primary=True. That way you can hide replied and keep all remaining of the arch as it is in mass mailing. Primary=True means a new view will be created (not applied as classic inheritance then) but using the parent arch + xpath to build its content.